File: youngtao-api/youngtao-provider-api/youngtao-provider-omc-api/src/main/java/com/youngtao/omc/api/constant/OrderStatus.java
package com.youngtao.omc.api.constant;

/**
 * @author ankoye@qq.com
 * @date 2020/12/13
 */
public class OrderStatus {

    /**
     * 已完成
     */
    public static final int COMPLETED = 0;

    /**
     * 创建中
     */
    public static final int CREATING = 1;

    /**
     * 待付款
     */
    public static final int PAYMENT = 2;

    /**
     * 待发货
     */
    public static final int DELIVERY = 3;

    /**
     * 待收货
     */
    public static final int RECEIVING = 4;

    /**
     * 待评价
     */
    public static final int COMMENT = 5;

    /**
     * 创建失败
     */
    public static final int FAILED = 9;

    /**
     * 退货中
     */
    public static final int RETURN = 11;

    /**
     * 已退款
     */
    public static final int REFUND = 12;

    /**
     * 关闭
     */
    public static final int CLOSE = 100;

}

File: youngtao-web/src/common/constant/OrderStatus.ts
/**
 * 已完成
 */
const COMPLETED = 0;

/**
 * 创建中
 */
const CREATING = 1;

/**
 * 待付款
 */
const PAYMENT = 2;

/**
 * 待发货
 */
const DELIVERY = 3;

/**
 * 待收货
 */
const RECEIVING = 4;

/**
 * 待评价
 */
const COMMENT = 5;

/**
 * 退货中
 */
const RETURN = 6;


/**
 * 创建失败
 */
const FAILED = 9;

/**
 * 关闭
 */
const CLOSE = 10;

export default {
	COMPLETED,
	CREATING,
	PAYMENT,
	RECEIVING,
	DELIVERY,
	COMMENT,
	RETURN,
	FAILED,
	CLOSE
}